setup.py

```diff
@@ -14,6 +14,7 @@
     'cssselect2',
     'tinycss2',
     'lxml',
+    'PyICU==1.9.8;platform_system=="Darwin"',  # FIXME link/symbol prob OSX
```

Comment:
These specifiers don't cascade real well. This may not be an issue if what you are installing is this distribution in particular rather than a distribution that depends on it.

Seems like you'd just want to pin this for everything rather than just mac. I'm thinking the distribution internals will treat this double dependency differently depending on circumstances.

Reply:
it's an order of operation thing. I tested in by using a new virtualenv and setting the platform dependency to "Linux" However, seems I was premature merging, since Phil can't get it to work?

Reply:
Why would we want to have unnecessary pins in the dev flow of the code? Thus is born technical debt

Reply:
It did do the down-grade as expected, when I had 2.0.3 installed and ran w/ `PyICU==1.9.8;platform_system="Linux"`